test/errors/useImportNotModule-detailed.good

```diff
@@ -0,0 +1,7 @@
+─── error in useImportNotModule.chpl:8 [UseImportNotModule] ───
+  Attempted to use 'someProc' which is not a module.
+  Target named here:
```

Comment:
In most errors, I just default to "In the following (whatever):". This helps keep them more standard, and avoids the need to introduce more terminology (e.g. "Target". I didn't know the thing-being-imported is a target!)